lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -27,9 +27,11 @@
       - "A string expression of the same form that can be passed to the 'when' statement"
       - "Alternatively, a list of string expressions"
     required: true
-  msg:
+  fail_msg:
     description:
       - "The customized message used for a failing assertion"
+    aliases:
+      - msg
```

Comment:
update in docs that fail_msg is 'new in 2.7'

Reply:
all right

Reply:
@bcoca version info added